appveyor.yml

```diff
@@ -23,7 +23,7 @@ test_script:
   - node --version
   - npm --version
   - yarn --version
-  - yarn run ci
+  - yarn run ci -- --runInBand
```

Comment:
Without this, the Windows builds have all sorts of impossible-to-debug race conditions with the filesystem operations (fixture copy, etc). _With_ it, it runs super fast (relative to "normal" multi-process Jest runs on Windows).